Gruntfile.js

```diff
@@ -1,120 +0,0 @@
-module.exports = function (grunt) {
-
-  grunt.initConfig({
-
-    watch: {
-      js: {
-        files: ['src/**/*.js'],
-        tasks: ['concat:app', 'babel', 'browserify', 'concat:lib', 'concat:dist'],
-        options: {
-          spawn: false,
-        },
-      },
-      css: {
-        files: ['src/main.scss'],
-        tasks: ['sass', 'concat:css', 'copy'],
-        options: {
-          spawn: false,
-        },
-      }
-    },
-
-    sass: {
-      dist: {
-        options: {
-          style: 'expanded',
-          sourcemap: 'none'
-        },
-        files: {
-          'dist/app.css': 'src/main.scss'
-        }
-      }
-    },
-
-    babel: {
-      options: {
-        sourceMap: true,
-        presets: ['es2015']
-      },
-      app: {
-        files: {
-          'dist/app.js': ['dist/app.js']
-        }
-      }
-    },
-
-    browserify: {
-      dist: {
-        files: {
-          'dist/app.js': 'dist/app.js'
-        }
-      }
-    },
-
-    concat: {
-      options: {
-        separator: ';',
-      },
-      app: {
-        src: [
-          'src/**/*.js',
-        ],
-        dest: 'dist/app.js',
-      },
-
-      lib: {
-        src: [
-          'node_modules/jquery/dist/jquery.min.js',
-          'node_modules/bootstrap/dist/js/bootstrap.min.js',
-          'node_modules/summernote/dist/summernote.min.js',
-          'node_modules/sn-components-api/dist/dist.js',
-          'vendor/modes/markdown/markdown.js'
```

Comment:
@mobitar `vendor` directory did not exist before this PR, but I'm wondering if this is the `CodeMirror` plugin for Markdown: https://codemirror.net/mode/markdown/

Reply:
Could have been a copy/paste from another editor repo..

Reply:
(I don't think this repo has anything to do with Markdown)

Reply:
That's what I thought, thanks 😄